src/app/util/ember-compatibility-functions.cpp

```diff
@@ -408,6 +405,15 @@ CHIP_ERROR GlobalAttributeReader::Read(const ConcreteReadAttributePath & aPath,
             }
             return CHIP_NO_ERROR;
         });
+    case EventList::Id:
+        return aEncoder.EncodeList([this](const auto & encoder) {
+            for (size_t i = 0; i < mCluster->eventCount; ++i)
+            {
+                ReturnErrorOnFailure(encoder.Encode(mCluster->eventList[i]));
+            }
+            return CHIP_NO_ERROR;
+        });
+        return CHIP_NO_ERROR;
```

Comment:
```suggestion
```

The previous line is a return (though kinda long line, effectively), so this would never be reached.